extensions-contrib/orc-extensions/src/test/java/io/druid/data/input/orc/OrcIndexGeneratorJobTest.java

```diff
@@ -291,8 +291,8 @@ private void verifyJob(IndexGeneratorJob job) throws IOException
         Assert.assertEquals(Integer.valueOf(9), dataSegment.getBinaryVersion());
 
         Assert.assertEquals(dataSourceName, dataSegment.getDataSource());
-        Assert.assertTrue(dataSegment.getDimensions().size() == 1);
-        String[] dimensions = dataSegment.getDimensions().toArray(new String[dataSegment.getDimensions().size()]);
+        Assert.assertEquals(1, dataSegment.getDimensions().size());
+        String[] dimensions = dataSegment.getDimensions().toArray(new String[0]);
```

Comment:
There is a corresponding inspection in IntelliJ, could you assign it ERROR severity (you could do it via IntelliJ interface; the change should appear in https://github.com/apache/incubator-druid/blob/master/.idea/inspectionProfiles/Druid.xml) so such problems are caught by CI.

Reply:
Yes. Indeed, in this way, such a similar work will be more effective. By the way, can we learn from Alibaba's Java coding specification project [P3C](https://github.com/alibaba/p3c). Although the specifications in it are not completely correct... For example, i just submitted [the issue about the pre-allocated size toArray method](https://github.com/alibaba/p3c/issues/348) to P3C a few days ago. :sweat_smile: